packages/@tinacms/toolkit/src/packages/react-sidebar/components/NoFormsPlaceHolder.tsx

```diff
@@ -31,11 +31,8 @@ export const NoFormsPlaceholder = () => (
       so you can start editing.
     </p>
     <p>
-      <LinkButton
-        href="https://tina.io/docs/forms/"
-        target="_blank"
-      >
-        <Emoji>📖</Emoji> Form Setup Guide
+      <LinkButton href="https://tina.io/docs/schema/" target="_blank">
+        <Emoji>📖</Emoji> Content Modelling
```

Comment:
```suggestion
        <Emoji>📖</Emoji> Content Modeling
```
Not sure 🤔 https://www.grammarly.com/blog/modeling-or-modelling/

Reply:
We've got a few changes to make if we're going with `Modeling` than `Modelling`...